src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -474,7 +474,7 @@ void primitive_inst::rebuild_exec_deps(
                 break;
             }
         }
-        OPENVINO_ASSERT(found, _exec_dep_ids[i], "not found in _exec_order");
+        OPENVINO_ASSERT(found, _exec_dep_ids[i], "not found in primitives while rebuilding _exec_deps");
```

Comment:
https://github.com/openvinotoolkit/openvino/pull/13801#discussion_r1017895716
I updated error messages here and other places. Thank you.